src/test/java/com/wikia/webdriver/common/contentpatterns/AdSlot.java

```diff
@@ -5,8 +5,8 @@
 public enum AdSlot {
   TOP_LEADERBOARD("top_leaderboard","div[id$='top_leaderboard_0__container__']"),
   BOTTOM_LEADERBOARD("bottom_leaderboard","div[id*='wka1a.PF/bottom_leaderboard'][id*='_0__container__']"),
-  TOP_BOXAD("top_boxad", "div[id$='top_boxad']"),
-  INCONTENT_BOXAD("incontent_boxad", "div[id$='incontent_boxad']"),
+  TOP_BOXAD("top_boxad", "div[id*='top_boxad_0__container__']"),
+  INCONTENT_BOXAD("incontent_boxad", "div[id*='HiVi/incontent_boxad']"),
```

Comment:
This selector looks strange. Where does `HiVi` come from? Do we really need it? :)

Reply:
Changed to /incontent_boxad